app/main/data_requests.py

```diff
@@ -20,7 +20,12 @@ def post_ingest(file: FileStorage, data: dict = None) -> tuple[bool, dict | None
     request_url = Config.DATA_STORE_API_HOST + "/ingest"
     files = {"excel_file": (file.name, file, MIMETYPE.XLSX)}
 
-    response = requests.post(request_url, files=files, data=data)
+    try:
+        response = requests.post(request_url, files=files, data=data)
+    except ConnectionError:
+        current_app.logger.error(f"Attempted POST to {request_url} but connection failed")
+        abort(500)
```

Comment:
do you also need to re-raise the error? Or is the abort enough?

Reply:
the error log will raise it in sentry and the abort will raise a 500 `HTTPError` exception and subsequently show the 500 error screen to the user